### PR TITLE
inspect: display buildkit daemon configuration file

### PR DESCRIPTION
**Pull request by crazy-max**
follow-up https://github.com/docker/buildx/issues/2683#issuecomment-2343001508

Displays buildkit daemon configuration file content used when creating a builder in inspect output:

```toml
# ~/.docker/buildx/buildkitd.default.toml

# debug enables additional debug logging
debug = true
# trace enables additional trace logging (very verbose, with potential performance impacts)
trace = true
# insecure-entitlements allows insecure entitlements, disabled by default.
insecure-entitlements = [ "network.host", "security.insecure" ]

[log]
  # log formatter: json or text
  format = "text"
```

```
$ docker buildx create --name builder --driver-opt "image=moby/buildkit:v0.16.0"
builder
```

```
$ docker buildx inspect builder --bootstrap
Name:          builder
Driver:        docker-container
Last Activity: 2024-09-11 08:31:27 +0000 UTC

Nodes:
Name:                  builder0
Endpoint:              unix:///var/run/docker.sock
Driver Options:        image="moby/buildkit:v0.16.0"
Status:                running
BuildKit daemon flags: --allow-insecure-entitlement=network.host
BuildKit version:      v0.16.0
Platforms:             linux/amd64, linux/amd64/v2, linux/amd64/v3, linux/arm64, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/mips64le, linux/mips64, linux/arm/v7, linux/arm/v6
Features:
 Automatically load images to the Docker Engine image store: false
 Cache export:                                               true
 Docker exporter:                                            true
 Multi-platform build:                                       true
 OCI exporter:                                               true
Labels:
 org.mobyproject.buildkit.worker.executor:         oci
 org.mobyproject.buildkit.worker.hostname:         e5772cf17e94
 org.mobyproject.buildkit.worker.network:          host
 org.mobyproject.buildkit.worker.oci.process-mode: sandbox
 org.mobyproject.buildkit.worker.selinux.enabled:  false
 org.mobyproject.buildkit.worker.snapshotter:      overlayfs
GC Policy rule#0:
 All:           false
 Filters:       type==source.local,type==exec.cachemount,type==source.git.checkout
 Keep Duration: 48h0m0s
 Keep Bytes:    488.3MiB
GC Policy rule#1:
 All:           false
 Keep Duration: 1440h0m0s
 Keep Bytes:    94.06GiB
GC Policy rule#2:
 All:        false
 Keep Bytes: 94.06GiB
GC Policy rule#3:
 All:        true
 Keep Bytes: 94.06GiB
File#buildkitd.toml:
 > debug = true
 > insecure-entitlements = ["network.host", "security.insecure"]
 > trace = true
 >
 > [log]
 >   format = "text"
 >
```

### CODE REVIEW

File: commands/inspect.go [hunk 1]
```diff
@@ -126,6 +126,14 @@ func runInspect(ctx context.Context, dockerCli command.Cli, in inspectOptions) e
 						fmt.Fprintf(w, "\tKeep Bytes:\t%s\n", units.BytesSize(float64(rule.KeepBytes)))
 					}
 				}
+				if debug.IsEnabled() {
```
Comment (crazy-max) [inline commands/inspect.go:129]: Not quite sure if we should do this only when debug is enabled :thinking:

Comment (tonistiigi) [reply commands/inspect.go:129]: I think if file is set, then just show it.

File: hack/test-driver [hunk 2]
```diff
@@ -105,7 +105,7 @@ if [ "$DRIVER" != "docker" ]; then
 fi
 
 # inspect and bootstrap
-buildxCmd inspect --bootstrap --builder="${builderName}"
+buildxCmd --debug inspect --bootstrap --builder="${builderName}"
```
Comment (tonistiigi) [inline hack/test-driver:108]: Why is this needed?

**Review by tonistiigi — APPROVED** [review]

**Comment by crazy-max** [comment]
needs follow-up on actions-toolkit repo: https://github.com/docker/actions-toolkit/blob/dd3654acca27188c34a78ee9270a5670056123dd/src/buildx/builder.ts#L85
